### ISSUE TITLE
'grunt export' doesnt export bower components dir

### ISSUE THREAD

**Issue by dilrajahdan** [north/generator-style-prototype#18]
i've managed to run grunt export... but I get the following issues:
- links to css are looking at /page/css - should be /css
- bower components dir has not been copied across
- links to js are looking at /page/components - not sure what this should be because I already have a dir called components which contains index.html of my components page.

any ideas??

thanks

**Comment by dilrajahdan** [comment]
Ok getting a bit further. Looks like 

`
          ,{
            expand: true,
            cwd: componentsDir,
            src: ['**'],
            dest: distPath + '/' + componentsDir
          }
`

was missing from the `dist:` bit in `guntfile.js`.

Running `grunt export` now copies the bower components over - awesome. Now I have to figure out how to get the scripts/css looking at the correct paths. I think its got something to do with `assetPrefix` in 'config.yml'.

**Comment by dilrajahdan** [comment]
Right - now paths are correct. 

I changed `assets: '/' + assetPrefix` to `assets: ''` in `Gruntfile.js` somewhere around line 138.

``` javascript
    generator: {
      dev: {
        files: [{
          cwd: pagesDir,
          src: ['**/*'],
          dest: root,
          ext: '.html'
        }],
        options: {
          partialsGlob: [partialsDir + '/{,**/}*.html', partialsDir + '/{,**/}*.md'],
          templates: templatesDir,
          handlebarsHelpers: helpers,
          userConfig: userConfig,
          environment: 'dev',
          development: true,
          lrport: lrport,
          assets: ''
        }
      },
      dist: {
        files: [{
          cwd: pagesDir,
          src: ['**/*'],
          dest: distPath,
          ext: '.html'
        }],
        options: {
          partialsGlob: [partialsDir + '/{,**/}*.html', partialsDir + '/{,**/}*.md'],
          templates: templatesDir,
          handlebarsHelpers: helpers,
          userConfig: userConfig,
          environment: 'prod',
          development: false,
          assets: ''
          //assets: '/' + assetPrefix

        }
```

Now I have to read the docs again - properly this time. I dont like my bower js components in the same place my index.html is for my components page.

**Comment by Snugug** [comment]
The current version of SP does export bower components correctly, however there were two bugs that made it appear a bit wonky. The first was that if you set `export.assetPrefix` in `config.yml` (the correct place to be editing your prefix) to `''`, the resultant URL generated would start with `//`, which is not what we want. The second was that, even if you had changed that, if you had run `grunt export`, got an `export` folder, and run it again, those files wouldn't be updated. This has been resolved by checking to see if the export path exists and, if it does, remove it before exporting.

This will be resolved later this morning in v0.6.1
